### PR TITLE
asdasd

### PR DESCRIPTION
**Pull request by VladimirCW**
fasdfasdfasdfasf
afasdfafafasdfasf
asdfsdasf

### CODE REVIEW

File: first.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ sdasdasdasdas
 A1
 sdfafsdff
 sdfasdf
-sfadfafsdfa
\ No newline at end of file
+sfadfafsdfa
+fsdfafasdfsd
```
Comment (VladimirCW) [inline first.txt:6]: delete this

File: first.txt [hunk 2]
```diff
@@ -2,4 +2,5 @@ sdasdasdasdas
 A1
 sdfafsdff
 sdfasdf
-sfadfafsdfa
```
Comment (kayurm) [inline first.txt:5]: comment*

File: first.txt (hunk 1 above)
Comment (Mjoee) [inline first.txt:6]: delete

File: first.txt [hunk 3]
```diff
@@ -2,4 +2,5 @@ sdasdasdasdas
...
\ No newline at end of file
+sfadfafsdfa
```
Comment (Misha842911) [inline first.txt:5]: delete

File: first.txt (hunk 1 above)
Comment (olga-kren) [inline first.txt:6]: test comment

File: hello.txt [hunk 4]
```diff
@@ -2,4 +2,5 @@ sdfsdfadf
 sdafsdfasdfasdfas
 asfsfasfsda
 sfsfasdfasf
-sdfasdfaf
\ No newline at end of file
+sdfasdfaf
+dfafasfds
```
Comment (Leonid-verif) [inline hello.txt:6]: good

**Review by olga-kren — APPROVED** [review]

**Review by kayurm — APPROVED** [review]

**Review by Mjoee — APPROVED** [review]

**Review by Leonid-verif — APPROVED** [review]

**Comment by Mjoee** [comment]
+

**Review by Mjoee — APPROVED** [review]

**Pull request merged**